tools/clang/test/CodeGenHLSL/quick-test/matrix_orientation_input.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: %dxc /T vs_6_0 /E main %s | FileCheck %s
+
+// Test reading input matrices in both orientations.
+
+int main(row_major int4x4 r : R, column_major int4x4 c : C) : OUT
+{
+    // Unfortunately the loads get reordered here.
```

Comment:
Should I split this into two tests? I don't like to depend on a seemingly random reordering.